src/libaktualizr/storage/sqlstorage.cc

```diff
@@ -1700,7 +1760,8 @@ boost::optional<std::pair<uintmax_t, std::string>> SQLStorage::checkTargetFile(c
 class SQLTargetWHandle : public StorageTargetWHandle {
  public:
   SQLTargetWHandle(const SQLStorage& storage, Uptane::Target target)
-      : db_(storage.dbPath()), target_(std::move(target)) {
+      : db_path_(storage.dbPath()), target_(std::move(target)) {
+    storage_ = &storage;
```

Comment:
`db_path` looks unused now, can it be removed? And I think `storage_` can be set in the initializer list.

Reply:
Moved storage_ to initializer list. The db_path_ is still using, it's mostly for existing test code.